test/Integration/Porter/Connector/CachingConnectorTest.php

```diff
@@ -41,6 +44,15 @@ public function testCacheDisabled()
         self::assertSame('bar', $this->connector->fetch('baz', $this->options));
     }
 
+    public function testCache()
+    {
+        /** @var MemoryCache $cache */
+        $cache = \Mockery::mock(MemoryCache::class);
```

Comment:
Please mock the interface (`CacheItemPoolInterface`) rather than an implementation. (`MemoryCache`).